src/main/java/dictionaryimplementation/Dictionary.java

```diff
@@ -0,0 +1,196 @@
+package dictionaryimplementation;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Scanner;
+import java.util.HashMap;
+import java.util.ArrayList;
+import java.lang.Math;
+import java.util.Set;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+
+public class Dictionary {
```

Comment:
you have already added the dictionary class in your previous pull request. Then why is this same file in the this pr also?